cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -405,18 +405,25 @@
     <name>audit.enabled</name>
     <value>true</value>
     <description>
-      Determines whether to publish audit messages to Apache Kafka
+      Determines whether to publish audit messages
     </description>
   </property>
 
   <property>
-    <name>audit.kafka.topic</name>
+    <name>audit.topic</name>
     <value>audit</value>
     <description>
-      Apache Kafka topic name to which audit messages are published
+      The topic name in the messaging system to which audit messages are published
```

Comment:
I think everything we have here are for the CDAP system settings, that's why I don't say it (as not to confuse user as well).